core/src/main/java/org/frankframework/http/rest/ApiServiceDispatcher.java

```diff
@@ -90,30 +72,46 @@ private List<ApiDispatchConfig>  findMatchingConfigsForUri(String uri, boolean e
 
 		for (Iterator<String> it = patternClients.keySet().iterator(); it.hasNext();) {
 			String uriPattern = it.next();
-			if(log.isTraceEnabled()) log.trace("comparing uri [{}] to pattern [{}]", uri, uriPattern);
+
+			if (log.isTraceEnabled()) log.trace("comparing uri [{}] to pattern [{}]", uri, uriPattern);
 
 			String[] patternSegments = uriPattern.split("/");
+
 			if (exactMatch && patternSegments.length != uriSegments.length || patternSegments.length < uriSegments.length) {
 				continue;
 			}
 
 			int matches = 0;
 			for (int i = 0; i < uriSegments.length; i++) {
-				if(patternSegments[i].equals(uriSegments[i]) || patternSegments[i].equals("*")) {
+				if (patternSegments[i].equals(uriSegments[i]) || patternSegments[i].equals("*")) {
 					matches++;
 				}
 			}
-			if(matches == uriSegments.length) {
+			if (matches == uriSegments.length) {
 				ApiDispatchConfig result = patternClients.get(uriPattern);
 				results.add(result);
 				if (exactMatch) {
 					return results;
 				}
 			}
+
+			if (uri.contains("**")) {
+				ApiDispatchConfig result = new ApiDispatchConfig(matchUri(uriPattern, uri));
+				results.add(result);
+				return results;
+			}
```

Comment:
Je moet nooit zelf een nieuw patroon registreren, dat is de taak van de `registerServiceClient` methode.